### PR TITLE
📝 Add documented examples for deposit

### PR DESCRIPTION
**Pull request by andreihrs**
### Checklist

- [] if I'm working with the latest farms.so version, I have modified the so with the latest one from master
- [] if I'm working with the latest kamino_lending.so version, I have modified the so with the latest one from master

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -72,3 +76,19 @@ yarn run reserve-caps
 ```bash
 yarn run reserve-supply-borrow
 ```
+
+
+### Deposit in reserve to mint ctokens
```
Comment (peroxy) [inline examples/README.md:81]: Please add some info to the docs that they need to setup a keypair file and their env variable for these.

File: examples/example_burn_ctokens_redeem_tokens.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import { getKeypair } from './utils/keypair';
+import { MAIN_MARKET, USDC_MINT } from './utils/constants';
+import BN from 'bn.js';
+import { TOKEN_PROGRAM_ID } from '@solana/spl-token';
+
+(async () => {
+  const connection = getConnection();
+  const wallet = getKeypair();
+
+  const market = (await KaminoMarket.load(connection, MAIN_MARKET, DEFAULT_RECENT_SLOT_DURATION_MS, PROGRAM_ID))!;
```
Comment (peroxy) [inline examples/example_burn_ctokens_redeem_tokens.ts:22]: Can you make a helper method that fetches the USDC reserve so you don't repeat code in all 3 examples?

**Review by peroxy — CHANGES_REQUESTED** [review]
few comments

File: examples/utils/keypair.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { Keypair } from '@solana/web3.js';
+import { getEnvOrThrow } from './env';
+
+export function getKeypair() {
+  const FILE_PATH = getEnvOrThrow('KEYPAIR_FILE');
+  return Keypair.fromSecretKey(Buffer.from(JSON.parse(require('fs').readFileSync(FILE_PATH))));
```
[the remark this answers is not in the document]
Comment (peroxy) [reply examples/utils/keypair.ts:6]: e.g.
```node
import { Keypair } from '@solana/web3.js';
import { readFileSync } from 'fs';

export function getKeypairFromFile(file: string): Keypair {
  const content = readFileSync(file);
  return Keypair.fromSecretKey(Uint8Array.from(JSON.parse(content.toString())));
}

```

**Review by peroxy — APPROVED** [review]

**Review by y2kappa — APPROVED** [review]

**Pull request merged**